src/three/math/Ellipsoid.d.ts

```diff
@@ -1,9 +1,5 @@
 import { Vector3, Matrix4 } from 'three';
 
-export const WGS84_RADIUS;
-export const WGS84_FLATTENING;
-export const WGS84_HEIGHT;
-
```

Comment:
Thanks for the catch - can we define a "constants.d.ts" file that exports these and add it to the index.d.ts file, as well?